examples/samplers/fast_sampling_with_jax_and_numba.ipynb

```diff
@@ -19,18 +19,58 @@
    "cell_type": "markdown",
```

Comment:
<p>I think it would be good to just run all these and make a little table with timings (compile, sample, wall, es/s)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Thought about that, but thought it would be clearer to have each model run in its own cell. Let me think about how best to do this.

Reply:
I also think every model should be run in it's own cell. But we could still collect timings as variables by doing `time.time()` before and after the call to sample. There also might be some profiler magic @ricardoV94 or @aseyboldt know to get the compile time vs the sampling time. My proposed method would just lump them together, which will bias against numba for example.

Reply:
<p>Lumping them together isn't terrible, as it will give an idea of timing in real-world usage. I guess I can increase the number of samples to more heavily weight sampling time.</p>